src/test/java/subway/application/PathServiceTest.java

```diff
@@ -0,0 +1,72 @@
+package subway.application;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.jdbc.Sql;
+import subway.application.dto.PathDto;
+import subway.domain.*;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static subway.application.PathServiceTest.Data.*;
+
+@ActiveProfiles("test")
+@Sql("/initialization.sql")
+@SpringBootTest
+class PathServiceTest {
+    @Autowired
+    private LineRepository lineRepository;
+
+    @Autowired
+    private StationRepository stationRepository;
+
+    @Autowired
+    private SectionRepository sectionRepository;
+
+    @Autowired
+    private PathService pathService;
+
+    @DisplayName("최단 경로 및 이용 요금을 조회할 수 있다.")
+    @Test
+    void test() {
+        //given
+        Long lineId1 = lineRepository.save(호선1);
+        Long lineId2 = lineRepository.save(호선2);
+
+        Long 회기Id = stationRepository.save(회기);
+        Long 청량리Id = stationRepository.save(청량리);
+        Long 왕십리Id = stationRepository.save(왕십리);
+        Long 잠실Id = stationRepository.save(잠실);
```

Comment:
```suggestion
        stationRepository.save(회기);
        Long 청량리Id = stationRepository.save(청량리);
        stationRepository.save(왕십리);
        Long 잠실Id = stationRepository.save(잠실);
```

회기Id, 왕십리Id는 사용하고 있지 않네요 ㅎㅎ

Reply:
통일성을 주고 싶었던 것 같습니다. 😭
하지만 오히려 혼란을 야기할 수도 있겠군요 !

해당 사항 반영하였습니다.